src/app/modules/shared_models/member_model.nim

```diff
@@ -137,12 +137,18 @@ QtObject:
     of ModelRole.LocalNickname:
       result = newQVariant(item.localNickname)
     of ModelRole.Alias:
+      if item.alias == "":
+        item.alias = singletonInstance.utils.generateAlias(item.pubKey)
       result = newQVariant(item.alias)
     of ModelRole.Icon:
       result = newQVariant(item.icon)
     of ModelRole.ColorId:
+      if item.colorId == 0:
+        item.colorId = singletonInstance.utils.getColorId(item.pubKey)
```

Comment:
So these are now getting properly initialized, but what about the other roles, won't they stay always empty?